### PR TITLE
优化SM2Util.load*FromFile方法中的字符流或字节流的关闭。

### PR DESCRIPTION
**Pull request by xiaohui249**
Signed-off-by: xiaohui249 <xiaohui_1123@126.com>

### CODE REVIEW

File: src/test/resources/pool-config.yaml [hunk 1]
```diff
@@ -6,11 +6,11 @@ default: &default
 
 sm2:
   <<: *default
-  maxTotal: 12
+  maxTotal: 30
 
 sm3:
   <<: *default
-  maxIdle: 10
+  maxIdle: 30
```
Comment (SamYuan1990) [inline src/test/resources/pool-config.yaml:13]: if maxIdle equals maxTotal then... means idle resource will not be released?

File: .gitignore [hunk 2]
```diff
@@ -6,4 +6,5 @@ build
 *.pem
 gradle/
 gradlew
-gradlew.bat
\ No newline at end of file
+gradlew.bat
+out/
```
Comment (SamYuan1990) [inline .gitignore:10]: what's in out folder?

File: src/test/resources/pool-config.yaml (hunk 1 above)
Comment (xiaohui249) [reply src/test/resources/pool-config.yaml:13]: It‘s wrong, I'll fix it. I want to set maxTotal 30 for the thread-safety test in parallel.

File: .gitignore (hunk 2 above)
Comment (xiaohui249) [reply .gitignore:10]: compile output files

**Review by SamYuan1990 — APPROVED** [review]
LGTM

**Pull request merged**